docs/src/aria-snapshots.md

```diff
@@ -329,10 +326,34 @@ When updating snapshots, Playwright creates patch files that capture differences
 applied, and committed to source control, allowing teams to track structural changes over time and ensure updates are
 consistent with application requirements.
 
+The source method for updating snapshots can be changed using the `--update-source-method` flag. There are several options available:
+
+- **"patch"** (default): Generates a unified diff file that can be applied to the source code using `git apply`.
+- **"3way"**: Generates merge conflict markers in your source code, allowing you to choose whether to accept changes.
+- **"overwrite"**: Overwrites the source code with the new snapshot values.
+
+```bash
+npx playwright test --update-snapshots --update-source-mode=3way
+```
+
+#### Snapshot as separate files
+
+To store your snapshots in a separate file, use the `toMatchAriaSnapshot` method with the `name` option, specifying a `.yml` file extension.
+
+```js
+await expect(page.getByRole('main')).toMatchAriaSnapshot({ name: 'main-snapshot.yml' });
+```
+
+By default, snapshots are saved in a directory next to your test file and a `.yml` file is created for each project specified in your Playwright configuration file. You can customize the snapshot path template using the `snapshotPathTemplate` option in the test runner configuration:
+
+```js
+snapshotPathTemplate: '__snapshots__/{testFilePath}/{arg}{ext}',
```

Comment:
Specify this in context of the config file, otherwise unclear where this property is.